game/hud/src/widgets/Welcome/index.tsx

```diff
@@ -0,0 +1,130 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import {client, events, faction, restAPI} from 'camelot-unchained';
+import * as React from 'react';
+
+export interface WelcomeProps {
+};
+
+export interface WelcomeState {
+  visible: boolean;
+  message: JSX.Element[];
```

Comment:
'dangerouslySetInnerHTML'  accepts regular html, so yes it is meant to use class= for= ect.